rpc/common/moc_common_common.proto

```diff
@@ -55,13 +55,21 @@ enum HealthState {
 message Error {
 	string Message = 1;
 	int32 Code = 2;
+	string Parameters = 3;
```

Comment:
what will this hold?

Reply:
comma separated, key value pairs which define parameters and values optionally encountered in the message, as defined by the idc team